src/d3a/models/area/__init__.py

```diff
@@ -192,8 +192,7 @@ def _cycle_markets(self, _trigger_event=True, _market_cycle=False, deactivate=Fa
     def tick(self, is_root_area=False):
         if ConstSettings.IAASettings.MARKET_TYPE == 2 or \
                 ConstSettings.IAASettings.MARKET_TYPE == 3:
-            for market in self._markets.markets.values():
-                market.match_offers_bids()
```

Comment:
Dont we still need this? And only publish_market_clearing when redis is enabled?

Reply:
You are totally right, we need this. What is more alarming here is why the integration tests pass despite this change. I will check it out.